.github/workflows/ci.yml

```
@@ -431,6 +431,25 @@ jobs:
       - name: Run dependency check
         run: ./ci/check_job_dependencies.sh
 
+  test-packing:
+    runs-on: ubuntu-latest
+    name: Checks Packing 
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - name: Testing Packing
```

A more descriptive name:
```suggestion
      - name: Check that LICENSE files are packaged
```